charts/rstudio-connect/Chart.yaml

```diff
@@ -1,8 +1,8 @@
 name: rstudio-connect
 description: Official Helm chart for RStudio Connect
-version: 0.2.9
+version: 0.2.10
 apiVersion: v2
-appVersion: 2021.10.0
+appVersion: 2021.11.0
```

Comment:
Will need to wait for the 2021.11.0 docker image to be available.

Reply:
Yes, merging this is blocked on waiting for Connect release 2021.11.0 to be available, which I'll add as a note within this (as well as blocked tag). Is there something else we'll need to do to get the 2021.11.0 docker image to be available?